docker-compose.yml

```diff
@@ -12,6 +12,14 @@ services:
       dockerfile: ./docker/Dockerfile.base
     image: local/pontoon_base
 
+  webapp.python3:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.py3.base
+      args:
+        - pythonversion=3.6.8
+    image: local/pontoon_base
```

Comment:
I think we can name this image `local/pontoon-py3` to be more explicit about its contents.